packages/client/components/ReflectionGroup/ReflectionGroup.tsx

```diff
@@ -239,7 +246,7 @@ const ReflectionGroup = (props: Props) => {
                   dataCy={`${dataCy}-card-${staticIdx}`}
                   key={reflection.id}
                   staticIdx={staticIdx}
-                  isClipped={staticIdx !== 0}
+                  isClipped={staticIdx > 0}
```

Comment:
It looks like `isClipped` sets the maxHeight of a reflection card to `44px` [here](https://github.com/ParabolInc/parabol/blob/750457f997fe1f3c233450f13bdb40c4fff0e460/packages/client/components/ReflectionEditorWrapper.tsx#L64). We want this to happen when there's a stack of reflections, e.g. if we group a small reflection on top of a large one, we want the large one to shrink/be clipped. 

I'm not sure when we would want to clip a card that has a `staticIdx` of -1? By only clipping cards that have an idx > 0, we prevent the space behind the animating remote reflection from shrinking to 44px. 

If we revert this change and Alice opens a very large reflection in the Spotlight, Bob will see reflections move into the empty space underneath the animating remote reflection.